test/e2e-go/features/catchup/basicCatchup_test.go

```diff
@@ -161,11 +162,16 @@ func runCatchupOverGossip(t *testing.T,
 
 		primaryStatus, err := lg.Status()
 		a.NoError(err)
-		a.True(nodeStatus.LastRound >= primaryStatus.LastRound)
-		if nodeStatus.LastRound == primaryStatus.LastRound && waitForRound < nodeStatus.LastRound {
+		if nodeStatus.LastRound <= primaryStatus.LastRound && waitForRound < nodeStatus.LastRound {
```

Comment:
Can't the secondary also advance and stay one round ahead of the primary and this condition never get satisfied?
 nodeStatus.LastRound <= primaryStatus.LastRound
Why not just keep one condition here: waitForRound < nodeStatus.LastRound

Reply:
The loop is checking the status every 50ms. We might momentarily be non-synchronized, but for the most of the time, the node's round would be synced. By writing `nodeStatus.LastRound <= primaryStatus.LastRound`, we'll be waiting until this condition is reached.